tox.ini

```diff
@@ -17,7 +16,7 @@ basepython =
 deps =
     -r{toxinidir}/requirements/test.txt
 commands =
-    py.test --cov=kolibri {posargs}
+    py.test --cov=kolibri --color=no {posargs}
```

Comment:
It's black text on black background using cmd.exe... :( Not everyone has a good terminal